packages/expo-symbols/src/SymbolView.tsx

```diff
@@ -2,11 +2,13 @@ import { Platform, requireNativeViewManager } from 'expo-modules-core';
 
 import { NativeSymbolViewProps, SymbolViewProps } from './SymbolModule.types';
 
-const NativeView: React.ComponentType<NativeSymbolViewProps> =
-  requireNativeViewManager('SymbolModule');
+let NativeView: React.ComponentType<NativeSymbolViewProps>;
+if (Platform.OS !== 'web') {
+  NativeView = requireNativeViewManager('SymbolModule');
+}
```

Comment:
Wouldn't it be more clear if moved to a `*.web.tsx` file?

Reply:
Maybe. There's not really any consistency to how we do this across the packages. A web file would just be 
```typescript
export function SymbolView(props: SymbolViewProps) {
    return props.fallback  
}
```

Reply:
Updated it